CHANGELOG.md

```diff
@@ -16,6 +16,13 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
     Security -- in case of vulnerabilities.
 -->
 
+## [1.0.1]
+
+### Added
+
```

Comment:
can we add updated progress and notification related item here !

Reply:
Good idea. I think I meant to but forgot.